### DISCUSSION TITLE
Evaluator Error caused by imageio-ffmpeg version

### DISCUSSION THREAD

**Discussion by ValkyrieMashiro** [PKU-Alignment/omnisafe#277] [General]
When I run evaluator, an error exists:
‘’‘
{
(omnisafe) xxx@ubuntu:~/projects_dir/omnisafe/examples$ python evaluate_saved_policy.p
##################################################
Saving the replay video to ./runs/PPOSaute-{SafetyPointGoal1-v0}/seed-5133-2023-09-07-14-37-12/video/epoch-300,
 and the result to ./runs/PPOSaute-{SafetyPointGoal1-v0}/seed-5133-2023-07-14-37-12/video/epoch-300/result.txt.
##################################################
Moviepy - Building video ./runs/PPOSaute-{SafetyPointGoal1-v0}/seed-5133-2023-09-07-14-37-12/video/epoch-300/eval-episode-0.mp4.
Moviepy - Writing video ./runs/PPOSaute-{SafetyPointGoal1-v0}/seed-5133-3-09-07-14-37-12/video/epoch-300/eval-episode-0.mp4

t:   0%|                                                                     | 0/1001 [00:00<?, ?it/s, now=None]
Traceback (most recent call last):
  File "/home/anaconda3/envs/omnisafe/lib/python3.10/site-packages/moviepy/video/io/ffmpeg_writer.py", li136, in write_frame
    self.proc.stdin.write(img_array.tobytes())
BrokenPipeError: [Errno 32] Broken pipe
}
’‘’

Then I found a solution originally posted by @novichiv in https://github.com/lengstrom/fast-style-transfer/issues/253#issuecomment-777344545_

''Workaround: downgrading imageio-ffmpeg version from 0.4.3 to 0.2.0 resolves this issue.''

And that bug disappear, the videos can be output correctly.

**Comment by muchvo** [comment]
Thank you very much for your feedback. We will verify this solution and consider addressing this dependency issue in the dependency configuration file or adding your feedback to a potential list of dependency issues.

**Comment by muchvo** [comment]
Also, if you are interested, you are very welcome to help us solve this potential dependency issue by submitting a PR as a contributor.
